lib/objectstream.js

```diff
@@ -2,8 +2,7 @@
 'use strict'
 
 var util = require('util');
-
-var EventEmitter = process.EventEmitter;
+var EventEmitter = require('events').EventEmitter;

```

Comment:
tested this change on node 8.9.1.  This module is a dependency of Appmetric.  Is there a plan to have this change merged?

Reply:
bump